resources/migrations/058_update_migrations.yaml

```diff
@@ -995,6 +995,211 @@ databaseChangeLog:
             path: instance_analytics_views/tables/v1/h2-tables.sql
             relativeToChangelogFile: true
 
+  - changeSet:
+      id: v58.2025-12-10T16:00:00
+      author: noahmoss
+      comment: Create metabot_use_case table
+      changes:
+        - createTable:
+            tableName: metabot_use_case
+            remarks: Use cases available for a metabot instance
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  remarks: Unique identifier for the use case
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: metabot_id
+                  type: int
+                  remarks: The metabot this use case belongs to
+                  constraints:
+                    nullable: false
+              - column:
+                  name: entity_id
+                  type: char(21)
+                  remarks: Random NanoID tag for entity
+                  constraints:
+                    nullable: true
```

Comment:
I see this is standard for `entity_id` but just for my own education why are these nullable?